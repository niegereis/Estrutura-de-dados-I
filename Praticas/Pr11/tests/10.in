10
6 2 5 3 2 4 1
Alice 1126786 30 52.8
Miguel 2529115 83 87.8
Sophia 2412560 79 99.1
Arthur 2529277 56 117.4
Helena 2926992 51 74.4
Bernardo 1625595 20 100.2
Valentina 2814037 87 81.7
Heitor 2328031 34 87.5
Laura 1816606 25 91.2
Davi 1210976 65 108.8
