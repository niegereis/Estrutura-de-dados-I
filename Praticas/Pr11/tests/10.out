[1625595] Bernardo - 20 (100.2)
[2926992] Helena - 51 (74.4)
[2412560] Sophia - 79 (99.1)
[2529277] Arthur - 56 (117.4)
[2328031] Heitor - 34 (87.5)
[2814037] Valentina - 87 (81.7)
[1816606] Laura - 25 (91.2)
[1210976] Davi - 65 (108.8)
[1126786] Alice - 30 (52.8)
[2529115] Miguel - 83 (87.8)
